### ISSUE TITLE
component=color|print in latex-image-preamble

### ISSUE THREAD

**Issue by Alex-Jordan** [APEXCalculus/APEXCalculusPTX#185]
I have a small concern about having multiple latex-image-preamble and using component=color|print to distinguish them. The main thing is that just now I discovered I had added content to one of them but I was oblivious to adding corresponding content to the other.

Can there just be one latex-image-preamble? And when building print, we have a latex.preamble.late that changes colors? It would not need to repeat all of the tikz setup commands. It could just redefine "firstcolor" to be black, and so on. If there is a need to create a few more semantic color names to get the grayscales just right, we could do that.

**Comment by sean-fitzpatrick** [comment]
That seems all right to me.
This was the best method I could think of, but if you can accomplish the same in the XSL, that works too.

But note that we also have two versions for `asymptote-preamble` and that seems harder (to me, at least) to work around, since we build Asymptote PDF prior to building LaTeX.

**Comment by Alex-Jordan** [comment]
Ah ok that's a good reason to keep components. I just realized we could
xi:include for the overlap. I'll try that with parse=text and see how it
goes, keeping the components.

**Comment by sean-fitzpatrick** [comment]
Sure. Improving on the current redundancy would be useful. I'm also thinking that edits to the docinfo aren't all that frequent, so having to double our efforts occasionally isn't too terrible.

**Comment by sean-fitzpatrick** [comment]
Should I leave this issue open while you test using `xi:include` for the duplicate content?

**Comment by Alex-Jordan** [comment]
Yes, let's keep this open as a reminder to try that. If it works, I can see lots of uses for this approach.

**Comment by sean-fitzpatrick** [comment]
@Alex-Jordan we're a month out from the anniversary of this issue. Can we close this, or are you still working on a better idea?

**Comment by sean-fitzpatrick** [comment]
This issue is now more than four years old, so I think it's safe to say it's not getting completed.
